pkg/operator/hive/hive_controller.go

```diff
@@ -337,6 +353,54 @@ func (r *ReconcileHiveConfig) Reconcile(request reconcile.Request) (reconcile.Re
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileHiveConfig) establishSecretWatch(hLog *log.Entry, hiveNSName string) error {
+	// We need to establish a watch on Secret in the Hive namespace, one time only. We do not know this namespace until
+	// we have a HiveConfig.
+	if !r.servingCertSecretWatchEstablished {
+		hLog.WithField("namespace", hiveNSName).Info("establishing watch on secrets in hive namespace")
+
+		// Create an informer that only listens to events in the OpenShift managed namespace
+		kubeInformerFactory := kubeinformers.NewSharedInformerFactoryWithOptions(r.kubeClient, 10*time.Minute,
```

Comment:
10 minutes seems like a very low resync period. Does that controller-runtime end up setting a larger resync period? Or will the hive-operator be resync HiveConfig every 10 minutes?

Reply:
I picked this up from the configmap watch for kube-apiserver-aggregator-client-ca above. I can bump them both to 30.